Makefile

```diff
@@ -6,19 +6,15 @@
 
 include dpf/Makefile.base.mk
 
-all: libs dgl plugins gen
+all: libs plugins gen
 
 # --------------------------------------------------------------
 
 libs:
 	$(MAKE) -C common
+	$(MAKE) -C dpf/dgl ../build/libdgl-opengl.a
 
-dgl:
-ifeq ($(HAVE_DGL),true)
-	$(MAKE) -C dpf/dgl
-endif
-
-plugins: libs dgl
+plugins: libs
```

Comment:
What is the rationale for editing the Makefile in the root directory? Does the `dgl` library conflict with something?